apps/site/pages/docs/getting-started/onboarding.mdx

```diff
@@ -0,0 +1,111 @@
+| Start Project | Opens the first page to start creating a new FastStore project. |
+
+## Starting a new FastStore project
+Now that you have an overview of the app, let's start the project.
+
+### Setp 1: Setup & Create Project
+
+1. Click on `Start Project`.
+2. In the **Setup & Create Project** page, ensure to define the basic configuration for your store:
+
+#### CMS Integration
+The option available to use a CMS in a FastStore project is [VTEX Headless CMS](/tbd), a VTEX'S Content Management solution.
+
+#### SEO Tags
+
+| Field | Description |
+| ------ | ----------- |
+| Title Tag | Type a keyword-optimized title for you store. For example: `Buy Online with the Best Game Store | GameStore.`. In this example, we used the length limit of around 50-60 characters to ensure it is fully displayed in search results without being truncated. It is also written in a way that accurately describes the webpage's content. |
+| Description Tag | Type a short description about your store, for example `Shop online at GameStore, the best game store for PC, Xbox, PS4 and Headphones. Explore our wide selection and buy your favorite games today!` |
+| Title Template | [TBD] For example `Headphones | GameStore`.|
+| Site Author| type the name of the agency or person responsible for the storefront. |
+
+#### Github Acesss
+After creating the FastStore project repository, you will need to gain access to its repository. In this section, you must enter the GitHub username of the person who will be the repository owner.
+
+#### Domain
+Type the current store URL.
+
+#### Google Tag Manager
+Type your store's Google Tag Manager ID (GTM ID) associated with a GTM container that manages and organizes your store's tags.
+
+#### Catalog
+Define which catalog your store needs to be linked with.
+
+3. Review the information you entered and ensure that it is correct.
+
+Once you have set up the basic configuration for your store, click on `Create` to proceed to the next page, `Deploy Summary`.
+
+### Step 2: Deploy Summary
+
+Once you have completed the setup for your store, the deploy process will start. 
+This process includes creating a repository and creating a live store. Once the process is finished, you will have one of the following scenarios:
+
+#### Repository and Live Store created
+
+![scenario-one-example](/tbd)
+
+In this scenario, both the repository and the live preview of the store were successfully created. You now have the following options:
+
+| Option | Description |
+| ------ | ----------- |
+| Your Repository | Access the store's Github repository and start setting up and customizing your store according to your brand.|
+| [Stores-Name] Preview| Preview your store's live demo. |
+| Documentation | Access the FastStore Portal Documentation.|
+| Storybook Components | Check out the native features and components. |
```

Comment:
should we link the `evergreen.faststore.dev` here?

Reply:
We don't necessarily need to add a link to this since it just describes the options available on the Admin's screen after the person successfully created the repository using the app. This table is based on a Figma file showing this scenario. So, maybe the `Storybook components`  option won't be there once launch the app 🤔